### DISCUSSION TITLE
Github Support Inacitve?

### DISCUSSION THREAD

**Discussion by plumpx** [community/community#41128] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I opened a ticket 2 days ago now and have not heard back from Support. My ticket is just a simple request regarding the Github Username Policy and I have not received a response. When I did research before making my request I seen other users make similar requests as mine and they got a response within 2-3 hours. Going on 2 days I want to make sure my request didnt get skipped over.

My ticket number is 1907673

The only reason I am concerned is due to my request after being answered is very time sensitive so I do not want to miss my opportunity if something happens. I am stuck checking it 2-4+ times a day.

**Comment by queenofcorgis** [comment]
Hi there - support times may vary depending on our team's resources, please be patient. The Community is unable to assist in escalating or responding to support tickets.
